Sources/JSON/JSON+Parse.swift

```diff
@@ -0,0 +1,693 @@
+/**
+ The MIT License (MIT)
+
+ Copyright (c) 2016 Ethan Jackwitz
+
+ Permission is hereby granted, free of charge, to any person obtaining a copy
+ of this software and associated documentation files (the "Software"), to deal
+ in the Software without restriction, including without limitation the rights
+ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ copies of the Software, and to permit persons to whom the Software is
+ furnished to do so, subject to the following conditions:
+
+ The above copyright notice and this permission notice shall be included in all
+ copies or substantial portions of the Software.
+
+ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ */
+
+import Core
+import libc
+
+extension JSON {
+    public convenience init(
+        serialized: Bytes,
+        skipNull: Bool = true,
+        allowFragments: Bool = true
+    ) throws {
+        var options: Parser.Option = []
+
+        if skipNull {
+            options.insert(.skipNull)
+        }
+
+        if allowFragments {
+            options.insert(.allowFragments)
+        }
+
+        let node = try JSON.Parser.parse(serialized, options: options)
+        self.init(_node: node)
+    }
+}
+
+
+// MARK: - JSON.Parser
+
+extension JSON {
+
+    public struct Parser {
+
+        public struct Option: OptionSet {
+            public init(rawValue: UInt8) { self.rawValue = rawValue }
+            public let rawValue: UInt8
+
+            /// Do not remove null values from the resulting JSON value. Instead store `JSON.null`
```

Comment:
Looks like I forgot to update my comment when I changed that flag from `.noSkipNull` to `.skipNull`.

`.skipNull` will actually leave `null` out of any objects or arrays. As such, the comment should be

```
/// Omit null values from `JSON.object`s & `JSON.array`s